tox.ini

```diff
@@ -30,22 +87,62 @@ skip_install = True
 deps =
     black==22.1.0
     click==8.0.2
-commands = black packages
+commands = black packages scripts tests
```

Comment:
You'll need a dummy test folder then

Reply:
Yes, I had it locally but I hadnt added an init.py so it wasnt pushed.